app/views/admins/_user_entry.haml

```diff
@@ -1,70 +1,87 @@
 
 %li.user.media
-  %div.pull-left
-    - if user.person
-      %span.media-object
-        = person_image_tag(user.person, size: :thumb_small)
+  .media-body.row
+    .col-md-1
+      - if user.person
+        %span.media-object
+          = person_image_tag(user.person, size: :thumb_small)
+    .col-md-11
+      .row
+        .col-md-7
+          %h4.media-heading
+            = user.person.name if user.person
```

Comment:
Use safe navigation (`&.`) instead of checking if an object exists before calling the method.